ChangeLog.txt

```diff
@@ -6,6 +6,8 @@ MAJOR:
 - Redesigned the dejure structure of Yemen and reworked Socotra
 - Reworked many colours across the map, based upon: https://forum.paradoxplaza.com/forum/index.php?threads/wtwsms-project-colouring-the-world.915304/
 - Overhauled the map of Mesopotamia and Western Persia, see on: http://forum.paradoxplaza.com/forum/index.php?threads/wtwsms-presentation-info-news-links.829142/page-4#post-21192710
+- Hypsistarians are no longer their own lonely group
+- Pagan players can now convert to Lucifarianism, Ebiniotoi, Antinomianism, and Egyptian Paganism
```

Comment:
That is intended functionality. Egyptian Paganism should stay like it currently is.

I'm not sure it's intended to be in in the first place, I think it's a left-over. The plan was to remove it.